### PR TITLE
New term Common.IsTimezone

### PR DESCRIPTION
**Pull request by HeikoTheissen**

### CODE REVIEW

File: vocabularies/Common.xml [hunk 1]
```diff
@@ -122,10 +122,15 @@
         <Annotation Term="Core.Description" String="The point in time represented by the annotated property or parameter shall be interpreted in the given time zone" />
         <Annotation Term="Core.LongDescription">
           <String>Time zones shall be specified according to the [IANA](https://www.iana.org/time-zones) standard.
-            If this annotation is absent or null or an empty string, points in time are typically interpreted in the current user's or default time zone.</String>
+            If this annotation is absent or null or an empty string, points in time are typically interpreted in the current user's or default time zone.
+            This annotation can be given as a path expression to a property tagged as [`IsTimezone`](#IsTimezone).</String>
```
Comment (ralfhandl) [inline vocabularies/Common.xml:126]:
```suggestion
            The annotation value can be a path expression resolving to a property tagged as [`IsTimezone`](#IsTimezone).</String>
```

**Review by ralfhandl — APPROVED** [review]
Looks good, maybe rephrase the description as suggested below

**Review by ralfhandl — APPROVED** [review]

File: vocabularies/Common.xml [hunk 2]
```diff
@@ -122,10 +122,15 @@
         <Annotation Term="Core.Description" String="The point in time represented by the annotated property or parameter shall be interpreted in the given time zone" />
         <Annotation Term="Core.LongDescription">
           <String>Time zones shall be specified according to the [IANA](https://www.iana.org/time-zones) standard.
-            If this annotation is absent or null or an empty string, points in time are typically interpreted in the current user's or default time zone.</String>
+            If this annotation is absent or null or an empty string, points in time are typically interpreted in the current user's or default time zone.
+            The annotation value can be a path expression resolving to a property tagged as [`IsTimezone`](#IsTimezone).</String>
```
Comment (uhlmannm) [inline vocabularies/Common.xml:126]: How has this to be interpreted? Is isTimezone mandatory for a property referenced in Timezone? And would the UI need to check also for the presence of isTimezone? Somehow that seems like overkill. How about "... resolving to a property that may be tagged as ..."

Comment (ralfhandl) [reply vocabularies/Common.xml:126]: Similar `Measures.Unit` and `Common.IsUnit`.

File: vocabularies/Common.xml [hunk 3]
```diff
@@ -122,10 +122,15 @@
         <Annotation Term="Core.Description" String="The point in time represented by the annotated property or parameter shall be interpreted in the given time zone" />
         <Annotation Term="Core.LongDescription">
           <String>Time zones shall be specified according to the [IANA](https://www.iana.org/time-zones) standard.
-            If this annotation is absent or null or an empty string, points in time are typically interpreted in the current user's or default time zone.</String>
+            If this annotation is absent or null or an empty string, points in time are typically interpreted in the current user's or default time zone.
+            The annotation value can be a path expression resolving to a property that may be tagged as [`IsTimezone`](#IsTimezone).</String>
```
Comment (ralfhandl) [inline vocabularies/Common.xml:126]:
```suggestion
            The annotation value can be a path expression resolving to a property that may be tagged with [`IsTimezone`](#IsTimezone).</String>
```

Comment (ralfhandl) [reply vocabularies/Common.xml:126]: In other vocabularies we consistently use "tagged with"

**Review by ralfhandl — APPROVED** [review]

**Review by uhlmannm — APPROVED** [review]
